docs/exploration-and-organization/delete-and-restore.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Delete and restore
+---
+
+# Delete and restore
+
+Sometimes your questions, dashboards, models, or collections outlive their usefulness. You can send outdated items to **Trash**.
+
+![Move to trash](./images/move-to-trash.png)
+
+Items in **Trash** won't show up in search (unless you use [advanced search filters](./exploration.md)), and you won't be able to use them to create new questions and dashboards.
+
+Moving items to Trash is not permanent: you'll be able to restore them to their original parent collection, or move them to another non-deleted collection. But if you'd' like to delete items permanently, [you can do that too](#permanently-deleting-items).
+
+Deleting an item will affect any [dashboards](../dashboards/introduction.md), [subscriptions](../dashboards/subscriptions.md), or [SQL questions](../questions/native-editor/referencing-saved-questions-in-queries.md) that depend on that item, so be careful!
+
+## See items in Trash
+
+You can find Trash at the bottom of the left navigation sidebar below all the collections:
+
+![Trash](./images/trash.png)
+
+You can think of Trash as a special type of collection. In **Trash**, you can see deleted items from the collections that you have [Curate permissions](../permissions/collections.md#collection-permission-levels) on. You can order deleted items by type (questions, dashboards, etc), time it was deleted, and who deleted it.
+
+You;'ll be able to see the contents of deleted dashboards, questions, and models in Trash, but you won't be able to modify them.
+
+## Search in Trash
+
+You can use [advanced search](./exploration.md) with a "Search items in trash" toggle to find items in Trash
+
+## Deleting and restoring items
+
+To move an item (question, dashboard, model, or collection) to Trash:
+
+1. Go to the question you want to delete;
+2. Click on the three dots menu;
+3. Select "Move to trash".
+
+> When a collection is deleted, all items in the collection are deleted as well.
+
+You'll still be able to see the contents of the items in Trash, but you won't be able to modify them or use them as a source for other questions.
+
+To restore an item:
+
+1. Go to Trash;
+2. Find the item you'd like to delete. You can sort deleted items to make it easier to find the item, or [search for your question in Trash](#search-in-trash);
+3. Click on the checkbox next to the item to select it;
+4. Select "Restore".
+
+> Restoring a collection will also restore all the items from that collection.
+
+If the item's original parent collection has been deleted as well, you won't see an option to **Restore**. You'll still be able to move the it from Trash to a different collection.
+
+## How deleting an item affects related items
+
+Deleting or restoring an item will affect other items that depend on that item.
+
+### Questions
+
```

Comment:
Maybe just a intro sentence for each table for more context? Something like:  "What happens to related items when you delete a question?"